charts/civil-sdt/Chart.yaml

```diff
@@ -8,7 +8,7 @@ maintainers:
   - name: HMCTS SDT team
 dependencies:
   - name: java
-    version: 4.0.11
+    version: 4.0.12
```

Comment:
Does the overall version property on line 5 also need to be incremented?  I've seen it done when changes are made to this file.

Reply:
Doing the absolute minimum here for the build to succeed i.e. what build said needed to be incremented. Not really part of this JIRA